dataeng/resources/tableau-backup.sh

```diff
@@ -1,30 +1,39 @@
 #!/usr/bin/env bash
-set -ex
 
 # This script ssh into Tableau server. Default centos user does not have access on Tableau directories
 # so shared folder is created that can be accessible from centos user. Backups are being created using
-# tsm maintenance backup utility. This script copies backup file into s3 bucket and then remove backup
-# file from tableau server.
+# tsm maintenance backup utility. This script creates the data and config backup and copies them into s3
+# bucket and then remove backup files from tableau server.
+
+ssh $USER_NAME@$TABLEAU_SERVER_IP /bin/bash << EOF
+set -ex
+sudo su - tsm_admin
+set -ex
+
+# Data backup filename in the format of table_data_backup_2022_01_16_T23_48Z.tsbak
+data_backup_filename="tableau_data_backup_$(date -u +%Y_%m_%d_T%H_%MZ).tsbak"
```

Comment:
Is it possible to stick with an ISO-compatible datetime format? https://en.wikipedia.org/wiki/ISO_8601

ISO 8601 allows the removal of non-alphanumeric characters, so this could be
```suggestion
data_backup_filename="tableau_data_backup_$(TZ=UTC; date -u +%Y%m%dT%H%MZ).tsbak"
```

which produces something that looks like this:

```
tableau_data_backup_20220817T1647Z.tsbak
```

You can actually confirm that this is valid parseable ISO format here: https://dencode.com/en/date/iso8601?v=20220817T1647Z&tz=UTC